src/stream/src/executor/aggregation/agg_group.rs

```diff
@@ -152,6 +151,29 @@ impl<S: StateStore> AggGroup<S> {
         Ok(())
     }
 
+    /// Write register state into state table for `AggState::Register`s
+    pub async fn sync_state(
```

Comment:
We finally have dirty flag again??🤣🤣

Can we just write the registers into state table on each `apply_chunk` or `get_output`? How much will that affect the performance?

Reply:
I remember that will introduce unnecessary encoding on modifying the memtable.

Reply:
That's to offload state table commitments to each `AggState` from hash/simple agg executors?

Reply:
> We finally have dirty flag again??🤣🤣
> 
> Can we just write the registers into state table on each `apply_chunk` or `get_output`? How much will that affect the performance?

just removed dirty flag

Reply:
> how about renaming it as "commit_state"

fixed